OrbitCore/Path.cpp

```diff
@@ -286,7 +286,7 @@ std::string Path::GetHome() {
   return home;
 }
 
-bool Path::ContainsFile(const std::string a_Dir, const std::string a_File) {
+bool Path::ContainsFile(const std::string& a_Dir, const std::string& a_File) {
```

Comment:
looks like the name of this method does not exactly reflect what it is actually doing.

Reply:
What do you suggest? `ContainsFilenameContaining`?

Reply:
It appears this function is used only once and the function that uses it can be simplified. #398